### DISCUSSION TITLE
AList使用指南

### DISCUSSION THREAD

**Discussion by icexmoon** [AlistGo/alist#2225] [Show and tell]
经过昨天和今天对AList的使用总结的，或许可以帮一些网友少走弯路，可以作为官方文档的补充：
https://blog.icexmoon.cn/archives/474.html

**Comment by wscqstu** [comment]
大神，我用笔记本挂载alist.exe，然后想用他的webdav在盒子里面使用这个webdav观看电影。
在IE里面测试webdav访问，http://IP address:5244/dav，弹出对话框，让你输入webdav的用户名和密码，这个用户名和密码是什么？？我输入系统的用户名和密码，提示是错误的。

**Reply by icexmoon** [reply]
默认是有一个admin和guest帐号的，admin是所有权限，你也可以根据需要添加一个用于webDAV的帐号，并指定根目录

**Comment by wscqstu** [comment]
我的版本是V3.4，我输入初始状态下的admin和密码，提示：Method Not Allowed，这个错误，是不是哪儿还需要设置呀？？

**Reply by icexmoon** [reply]
你应该检查客户端配置是否正确，比如某些客户端，如Windows自带的remote client，默认情况下连接webDAV服务只支持https连接，要通过http连接就需要修改注册表。换言之，你应该检查当前特定平台的客户端连接webDAV的配置是否正确，以及是否有某些特定要求。
